### ISSUE TITLE
haskellPackages.HsYAML-aeson failed to build on 20.03

### ISSUE THREAD

**Issue by poscat0x04** [NixOS/nixpkgs#83176]
**Describe the bug**
```
Setup: Encountered missing dependencies:
HsYAML >=0.2.0 && <0.3
```

**To Reproduce**
1. ...
2. ...
3. ...

**Expected behavior**

**Screenshots**

**Additional context**

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by cdepillabout** [comment]
This appears to be working on the current `haskell-updates` branch.

`HsYAML-aeson` probably just needs some overrides to either be jailbroken or use a different version of `HsYAML`.

Please feel free to send a PR to the `release-20.03` branch fixing this.

**Comment by cdepillabout** [comment]
Closing since there doesn't seem to be anyone working on this.

Please feel free to re-open if you'd still like this fixed for 20.03.
